build/Dockerfile.amd64

```diff
@@ -22,20 +23,13 @@ LABEL name="Calico Operator" \
       description="Calico Operator manages the lifecycle of a Calico or Tigera Secure installation on Kubernetes or OpenShift" \
       maintainer="Laurence Man <laurence@tigera.io>"
 
-# Add in top-level license file
-RUN mkdir /licenses
-COPY LICENSE /licenses
```

Comment:
I think license is still required for certification requirements. Sorry if I wasn't clear in my previous comment. My comment was to only remove /etc/passwd which shouldn't be required anymore.